libraries/Microsoft.Bot.Builder.AI.LUIS/LuisExtensions.cs

```diff
@@ -32,6 +32,10 @@ public static IConfigurationBuilder UseLuisSettings(this IConfigurationBuilder b
             settings["luis:endpoint"] = $"https://{luisRegion}.api.cognitive.microsoft.com";
             settings["BotRoot"] = botRoot;
             builder.AddInMemoryCollection(settings);
+            if (environment == "Development")
+            {
+                environment = Environment.UserName;
+            }
```

Comment:
Why the environment is set to UserName when it equals to "Development"?

Reply:
Because in development mode we want to tie to the individual.

---
In reply to: [402915990](https://github.com/microsoft/botbuilder-dotnet/pull/3669#discussion_r402915990) [](ancestors = 402915990)